pkg/cloud/buckets/buckets.go

```diff
@@ -46,29 +47,36 @@ func KubeProviderToBucketScheme(provider string) string {
 
 // ReadURL reads the given URL from either a http/https endpoint or a bucket URL path.
 // if specified the httpFn is a function which can append the user/password or token if using a git provider
```

Comment:
Might be worth updating the godocs here.

Reply:
Done.